src/components/molecules/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -11,26 +12,50 @@ interface Props {
   items: BreadcrumbItem[];
 }
 
+const styles: Styles<
+  'breadcrumb' | 'root' | 'separator',
+  Record<string, any>
+> = theme => ({
+  breadcrumb: {
+    'color': theme.colors.fgMuted[6],
+    '&:hover': {
+      color: theme.colors.indigo,
+    },
+  },
+});
+
+const concatPathLevels = (items: BreadcrumbItem[]) => {
+  const localItems: BreadcrumbItem[] = [];
+
+  items.map((item, level) => {
+    if (level === 0) return localItems.push(item);
+    return localItems.push({
+      ...item,
+      path: localItems[level - 1]?.path + item.path,
+    });
+  });
+
+  return localItems;
+};
```

Comment:
what do you think of the reduce version of it

```suggestion
const concatPathLevels = (items: BreadcrumbItem[]) => {
  return items.reduce<BreadcrumbItem[]>((prev, curr) => {
    if (prev.length === 0) return [curr];
    const path = prev[prev.length - 1]!.path + curr.path;
    return prev.concat({ ...curr, path });
  }, []);
};
```